Demo/Sources/Players/PlaybackView.swift

```diff
@@ -108,12 +108,16 @@ private struct MainView: View {
     private func main() -> some View {
         ZStack {
             video()
+                .accessibilityElement()
+                .accessibilityLabel("Video")
+                .accessibilityHint("Double tap to toggle controls")
+                .accessibilityAction(.default, visibilityTracker.toggle)
```

Comment:
Instead of having this hint and this action, should we keep controls always visible when **VoiceOver** is on?

Reply:
The fact that controls are persistent but togglable when VoiceOver is on is a desired feature. We cannot assume that only blind users use VoiceOver. People who still can see must be able to use the player, and they might want to have controls off so that they can still see the image. This is also how Apple standard player UI behaves.